rfcs/20181116-saved-model.md

```diff
@@ -0,0 +1,816 @@
+﻿# SavedModel Save/Load in 2.x
+
+| Status        | Proposed       |
```

Comment:
Can you please change the status to accepted, and then I will merge it.

Reply:
Done, thanks.